src/platform/site-wide/user-nav/sass/user-nav.scss

```diff
@@ -115,6 +116,7 @@ span.sidelines {
 
     &.idme-button {
       background: $color-green;
+      padding: 1em 0.75em;
```

Comment:
Do we have design system classes for this? https://design.va.gov/utilities/padding